### ISSUE TITLE
Add macOS Sierra runtime identifier

### ISSUE THREAD

**Issue by peterblazejewicz** [dotnet/templating#290]
The readme stipulates to check runtime identifier - if my runtime is being supported. The `macOS Sierra` runtime id is missing:
https://github.com/dotnet/templating/blob/master/src/dotnet-new3/dotnet-new3.csproj#L7
though `dotnet` supports it as `RID:         osx.10.12-x64`

Thanks!

**Comment by mlorbetske** [comment]
Trying this out - it looks like deleting that line in the project will fix it. Could you try it out to confirm (worked on Windows)? If so, I'll remove it from the repo

**Comment by peterblazejewicz** [comment]
@mlorbetske Mike, don't know, i"ve got a bunch of errors when running `setup.sh`:
https://gist.github.com/peterblazejewicz/b20186cbcd37adab2ad8fd88a1434ef7
The `dotnet new` itself seems to work fine
How I could check version of symlinked `dotnet new`?

**Comment by mlorbetske** [comment]
@peterblazejewicz it looks like `setup.sh`, `build.sh` and `harderreset.sh` have gotten out of date with some of the build changes we've needed to make lately. I'm working through resolving the issues with it.

**Comment by mlorbetske** [comment]
This is fixed with #401

**Comment by peterblazejewicz** [comment]
FYI:
```bash
Done!
You can now use setup.sh from anywhere to rebuild dotnet new3
You can now use harderreset.sh from anywhere to delete dotnet new3 artifacts
sed: 1: "defaultinstall.package. ...": extra characters at the end of d command
sed: 1: "defaultinstall.package. ...": extra characters at the end of d command
dotnet new3 is ready!
```
Thanks!
